tools/gen-wire-tests/main.go

```diff
@@ -55,6 +56,14 @@ var (
 	microk8s = Cloud{CloudName: "microk8s", ProviderName: "k8s"}
 )
 
+var minVersionRegex = map[string]string{
+	"3.0": "^[4-9].*|^3\\\\.([0-9]|\\\\d{2,})(\\\\.|-).*",
+	"3.1": "^[4-9].*|^3\\\\.([1-9]|\\\\d{2,})(\\\\.|-).*",
+	"3.2": "^[4-9].*|^3\\\\.([2-9]|\\\\d{2,})(\\\\.|-).*",
+	"3.3": "^[4-9].*|^3\\\\.([3-9]|\\\\d{2,})(\\\\.|-).*",
+	"3.4": "^[4-9].*|^3\\\\.([4-9]|\\\\d{2,})(\\\\.|-).*",
```

Comment:
You definitely need to add a comment about, what this is doing. Also, I'm not sure you need `[4-9].*|` there. We're some way away from releasing a 4. version and I hope we're not using this by then. Removing that will make the regex cleaner and easier to read, esp. with all the back slashes.

Reply:
> I suspect that is wishful thinking

We have no option, mutlijob is not supported, and we _need_ or __require__ to get off it.